examples/seismic/tti/wavesolver.py

```diff
@@ -37,6 +37,13 @@ def op_fwd(self, kernel='shifted', save=False):
                                space_order=self.space_order,
                                kernel=kernel, **self._kwargs)
 
+    @memoized_meth
+    def op_fwd_stagg(self, save=False):
```

Comment:
I'm now thinking whether the staggered tti should be entirely moved to a different module. 
In theory one could have 10 formulations of tti -- and what would we do, would we put all of them here? not to talk about adjoint etc

Reply:
Maybe we can discuss it. Bot are TTI so I left it under Aniso..Solver, but we may want to reorganize it.